dataplane/standalone/proto/common.proto

```diff
@@ -260,9 +260,19 @@ message ObjectTypeQueryResponse {
   ObjectType type = 1;
 }
 
+message InitializeRequest {}
+
+message InitializeResponse {}
+
+message UninitializeRequest {}
+
+message UninitializeResponse {}
+
```

Comment:
Are these being used?

Reply:
not currently. for reset need to figure out if we need to start and stop in a specific order.